src/index.js

```diff
@@ -0,0 +1,102 @@
+const registerEvents = () => {
+  console.log('loaded');
+
+  let currentTemp = 71;
+
+  const resetButton = document.getElementById('cityNameResetButton');
+  resetButton.addEventListener('click', () => {
+    document.getElementById('cityNameInput').value = 'Seattle';
```

Comment:
Rather than retrieving the UI elements each time a function runs, we could retrieve them once, ahead of any of the functions (as with `resetButton`), and then refer to those variables within the various functions.